lib/bolt/target.rb

```diff
@@ -80,6 +80,11 @@ def resources
       inventory_target.resources
     end
 
+    def resource(type, title)
+      rr = ResourceInstance.new({ 'type' => type, 'title' => title }).reference
```

Comment:
I feel like creating a new ResourceInstance here isn't the right approach. Maybe a class method that formats the type and title for you, which can be called from `ResourceInstance.reference` and `Target.resource`? Something like

```
def self.format_reference(type, title)
    "#{type}[#{title}]"
end

def reference
    self.class.format_reference(@type, @title)
end
```